cfme/base/ssui.py

```diff
@@ -145,7 +144,7 @@ def login(self, user=None, method=LOGIN_METHODS[-1]):
     time.sleep(3)
     login_view.flash.assert_no_error()
     self.browser.plugin.ensure_page_safe()
-    assert logged_in_view.is_displayed
+    logged_in_view.wait_displayed('10s')
```

Comment:
you can assert for this too .
assert logged_in_view.wait_displayed('10s')

Reply:
@sshveta I think, no need to assert... It will raise `Timeout` error. If fails to land within `10s` after a call.